capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java

```diff
@@ -657,6 +659,19 @@ private void startHelpActivity() {
         trackCameraScreenEvent(CameraScreenEvent.HELP);
     }
 
+    private void initOnlyQRScanning() {
+        if (isOnlyQRCodeScanning()) {
+            mPaneWrapper.setVisibility(View.GONE);
+        }
+    }
+
+    private boolean isOnlyQRCodeScanning() {
+        if (!GiniCapture.hasInstance()) {
+            return false;
+        }
+
+        return GiniCapture.getInstance().isQRCodeScanningEnabled();
```

Comment:
[Request] You need to add a new method to the `GiniCapture.Builder` to enable only QR code scanning mode. 

Then `isOnlyQRCodeScanning()` should return true if QR code scanning AND QR code only mode is enabled.